composeApp/src/commonMain/kotlin/io/github/couchtracker/db/common/adapters/DbIconColumnAdapter.kt

```diff
@@ -0,0 +1,9 @@
+package io.github.couchtracker.db.common.adapters
+
+import app.cash.sqldelight.ColumnAdapter
+import io.github.couchtracker.db.user.model.icon.DbIcon
+
+val DbIconColumnAdapter: ColumnAdapter<DbIcon, String> = CouchTrackerUriColumnAdapter.map(
```

Comment:
Question: will this always be a couch tracker uri? Will we support custom URL or hardcoded SVGs?

Reply:
The intention was to always have a `couch-tracker://` URI that can evolve as needed. For custom SVGs, for example, we could have this format: `couch-tracker://icon/svg/<base64>`